File: flask-rest-api/flask-rest-api/flask_rest_api/person.py
from flask_rest_api import app, db

class Person(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    first_name = db.Column(db.String(45))
    last_name = db.Column(db.String(45))
    email = db.Column(db.String(45))
    phone_number = db.Column(db.String(45))
    personal_number = db.Column(db.String(45))

    def __init__(self, first_name, last_name, email, phone_number, personal_number):
        self.first_name = first_name
        self.last_name = last_name
        self.email = email
        self.phone_number = phone_number
        self.personal_number = personal_number

    def __repr__(self):
        return '<Person %r>' % self.first_name

File: express_rest_api/express_rest_api/models/person.js
"use strict";

module.exports = function (sequelize, DataTypes) {
    var Person = sequelize.define("person", {
        id: {
            type: DataTypes.INTEGER,
            primaryKey: true,
            allowNull: false,
        },
        first_name: DataTypes.STRING,
        last_name: DataTypes.STRING,
        email: DataTypes.STRING,
        phone_number: DataTypes.STRING,
        personal_number: DataTypes.STRING
    }, {
            timestamps: false,
            tableName: 'person'
        });
    return Person;
};